rackcheck/info.rkt

```diff
@@ -3,7 +3,8 @@
 (define version "2.0.1")
 (define collection "rackcheck")
 (define deps '("base"
-               "rackunit-lib"))
+               "rackunit-lib"
+               "rackcheck-lib"))
 (define build-deps '("racket-doc"
                      "rackunit-doc"
                      "rackunit-lib"
```

Comment:
This needs an `update-implies` field like this

```racket
(define update-implies '("rackcheck-lib"))
```

so that when someone runs `raco pkg update rackcheck`, `rackcheck-lib` is also updated.

Reply:
Added update-implies https://github.com/Bogdanp/rackcheck/pull/10/commits/6ed370ce16f06018b627c3a6e76e33480c9154f5